packages/react-native/src/evm/components/ConnectWallet.tsx

```diff
@@ -69,6 +69,13 @@ export type ConnectWalletProps = {
    * ```
    */
   displayBalanceToken?: Record<number, string>;
+
+  /**
+   * Hide the "switch to Personal wallet" option in the wallet modal which is shown when wallet is connected to either Smart Wallet or Safe
```

Comment:
Please, don't mention Safe in the comments here since it's not available in RN :)